woocommerce/payment-gateway/External_Checkout/Google_Pay/Admin.php

```diff
@@ -99,7 +99,7 @@ public function get_settings() {
 			[
 				'id'      => 'sv_wc_google_pay_display_locations',
 				/* translators: Allow Google Pay button on selected display locations (e.g. cart, checkout, product page...) */
-				'title'   => __( 'Allow Google Pay on', 'woocommerce-plugin-framework' ),
+				'title'   => _x( 'Allow Google Pay on', 'Setting title for a multi-select list dropdown of display locations', 'woocommerce-plugin-framework' ),
```

Comment:
the translator didn't understand the context, but it's possible that they were looking at a file that was missing the translators comment